csrankings-x.csv

```diff
@@ -289,6 +289,7 @@ Xu Yu,Chinese University of Hong Kong,http://www.se.cuhk.edu.hk/people/yu.html,i
 Xu Zhu 0001,University of Liverpool,https://liverpool.ac.uk/electrical-engineering-and-electronics/staff/xu-zhu,NycCVyoAAAAJ
 Xuan Guo,University of North Texas,http://www.cse.unt.edu/~xuanguo,4tqv2FYAAAAJ
 Xuan Wang 0002,Harbin Institute of Technology,http://homepage.hit.edu.cn/wangxuan,NOSCHOLARPAGE
+Xuan Yang, Shenzhen University, http://csse.szu.edu.cn/cn/people?14711, W4JClUYAAAAJ

```

Comment:
Web page does not work. Removes spaces after commas. Google Scholar link incorrect. Please try again.